src/core/exec-invoke.c

```diff
@@ -5340,6 +5416,14 @@ int exec_invoke(
                 }
         }
 
+        if (context->private_bpf) {
+                r = bpffs_prepare(&bpffs_fd, &bpffs_pid);
+                if (r < 0) {
+                        *exit_status = EXIT_BPF;
+                        return log_error_errno(r, "Failed to mount BPFFS: %m");
```

Comment:
`BPFFS` -> `bpffs` ?
The error message is inconsistent with the one for `bpffs_finalize()`.